test/werewolf/engine/state_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+module Werewolf
+  module Engine
+    class StateTest < Minitest::Test
+      def test_starts_at_correct_state
+        assert_equal State.new.current, State::STATES.first
```

Comment:
you've got the args backwards on all these:  `assert_equal expected_value, actual_value`